src/tests/test_kube_api.py

```diff
@@ -83,14 +75,344 @@ def _get_vips(cls, cluster_config, nodes: Nodes):
 
         return api_vip, ingress_vip
 
+    @JunitTestSuite()
+    @pytest.mark.kube_api
+    def test_kube_api_ipv4(self, kube_test_configs_single_node, kube_api_context, get_nodes):
+        namespace = global_variables.spoke_namespace
+        cluster_config, tf_config = kube_test_configs_single_node
+        self.kube_api_test(kube_api_context.api_client, get_nodes(tf_config, cluster_config), cluster_config, namespace)
+
+    @JunitTestSuite()
+    @pytest.mark.kube_api
+    def test_kube_api_ipv6(self, kube_test_configs_single_node, kube_api_context, proxy_server, get_nodes):
+        cluster_config, tf_config = kube_test_configs_single_node
+        tf_config.is_ipv6 = True
+        tf_config.is_ipv4 = False
+
+        self.kube_api_test(
+            kube_api_context.api_client,
+            get_nodes(tf_config, cluster_config),
+            cluster_config,
+            global_variables.spoke_namespace,
+            proxy_server,
+            is_ipv4=False,
+        )
+
+    @classmethod
+    def start_nodes(cls, nodes: Nodes, infra_env: InfraEnv, entity_config: BaseEntityConfig, is_ipv4) -> List[Agent]:
+        infra_env.status()
```

Comment:
What this does?
And if it just to wait for the status, we should add some comment about that

Reply:
From the function  documentation:


```
        Status is a section in the CRD that is created after registration to
        assisted service and it defines the observed state of InfraEnv.
        Since the status key is created only after resource is processed by the
        controller in the service, it might take a few seconds before appears.
```

Reply:
Just like you did here: https://github.com/openshift/assisted-test-infra/blob/2581b64f4ce4b095f2dafd65d6d7ccf34d462c25/discovery-infra/test_infra/helper_classes/kube_helpers/infraenv.py#L282-L284

In the scope of this file it's a weird looking line of code that seems to have no side-effect

Reply:
You are giving me too much credit, that was there when I arrived :)